docs/appendix.rst

```diff
@@ -524,69 +509,104 @@ To fix this, install the Python "wheel" package::
-to just OpenSSL and possibly libreSSL.
+This error means that the WBEM server side SSL implementation does not yet support
+TLS 1.2 or higher or that the SSL library (OpenSSL or LibreSSL) used
+by pywbem does not support TLS 1.2 and the other side requires TLS GE 1.2.
 
-This also happened after an upgrade of the client OS to Debian buster
+See the Python document
+`https://peps.python.org/pep-0644/ <PEP 644 – Require OpenSSL 1.1.1 or newer>`_
+for more information on Python and OpenSSL version 1.1.1.
+
+In pywbem version 1.7.0 the urllib3 configuration version limit was modified to
+allow urllib3 package versions GE 2.0.  These new versions of urllib3 include backward
+incompatible changes including:
+
+* Support limiting the minimum TLS version to >= 1.2 (i.e OpenSSL version >= 1.1.1). If
+  the version of OpenSSL is less than 1.1.1, this SSLError will occur with the
+  initial request to the WBEM Server.
+* Limit the SSL library implementations to just OpenSSL and some versions of
+  LibreSSL.  The  version 2.0+ urllib3 implementation does not support any
+  other implementations of SSL.
+
+This also happens after an upgrade of the client OS to Debian buster
 using Python 3.7, with OpenSSL 1.1.1d.
 
-This is an error that is created by the OpenSSL library and handed back up to
-the SSL module of Python which hands it up to pywbem. The error indicates that
-OpenSSL and the WBEM server do not agree about which SSL/TLS protocol level to
-use.
+This is an error that is created by the SSL library (normally OpenSSL or
+LibreSSL) and handed back up to the SSL module of Python which hands it up to
+pywbem. The error indicates that OpenSSL and the WBEM server do not agree about
+which SSL/TLS protocol level to use.
 
 Pywbem specifies SSL parameters such that the highest SSL/TLS protocol version
-is used that both the client and server support. Thus, pywbem does not put any
-additional restrictions on top of OpenSSL.
+is used that both the client and server support. Pywbem itself does not put any
+additional TLS version restrictions on top of SSL library.
 
 Debian buster includes OpenSSL 1.1.1d and increased its security settings to
 require at least TLS 1.2 (see https://stackoverflow.com/a/53065682/1424462).
 
-This error means that the WBEM server side does not yet support
-TLS 1.2 or higher or that the SSL library used by pywbem does not support
-TL 1.2.
-
-This issue can be corrected by:
+This issue can possibly be corrected by:
 
 1. If the current version of urllib3 is less than 2.0, update the version
-   of urllib3 (ex. ``pip install --upgrade  --upgrade-strategy eager urllib3``). Since
-   the pywbem install does not force an eager update of packages, if a valid
-   previous version of urllib3 exists it will not be upgraded to 2.0+ in the
-   reinstallation of pywbem.
+   of urllib3 (ex. ``pip install --upgrade  --upgrade-strategy eager urllib3``).
+   Since the pywbem install does not force an eager update of packages, if a
```

Comment:
This sentence would leave me as a user a bit unsure about whether it states something I need to act upon. What it really means is to give the reason why urllib needs to be upgraded explicitly. I suggest to rephrase the sentence to clarify that it gives the reason for that, e.g. by starting it with "This is needed because..."